### PR TITLE
Add Tuya LH-961ZB motion sensor support

### PR DESCRIPTION
**Pull request by Tagnard**
This quirk fixes the problem there the iaszone state is never reset.

### CODE REVIEW

**Review by javicalle — CHANGES_REQUESTED** [review]
It seems to me a difficult device to classify within the Tuya ecosystem.
In any case the name of the quirk would change it to `SM0202Motion`

File: zhaquirks/tuya/sm0202_motion.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+            Union[t.Addressing.Group, t.Addressing.IEEE, t.Addressing.NWK]
+        ] = None
+    ):
+        """Handle cluster request."""
+        arg = args[0]
+        if arg & IasZone.ZoneStatus.Alarm_1:
+            """Handle the cluster command."""
+            if self._timer_handle:
+                self._timer_handle.cancel()
+            self._timer_handle = self._loop.call_later(self.reset_s, self._turn_off)
```
[the remark this answers is not in the document]
Comment (Tagnard) [reply zhaquirks/tuya/sm0202_motion.py:62]: I am new to building quirks and I didn't know what is did more than I used it as base for mine. 

After changing the  cluster to the one you showed me the sensor still works. Now I want to override the 30 second timer with a 60 second one, how do you do that using this predefined cluster? 

The motion sensor has a 60 second timeout before being able to trigger again.

**Comment by Tagnard** [comment]
Thanks for the information, I have updated it

**Review by javicalle — APPROVED** [review]
LGTM

File: zhaquirks/tuya/sm0202_motion.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+    reset_s: int = 60
+
+
+class SM0202Motion(CustomDevice):
+    """Quirk for LH-961ZB motion sensor."""
+
+    def __init__(self, *args, **kwargs):
+        """Init."""
+        super().__init__(*args, **kwargs)
```
Comment (TheJulianJES) [inline zhaquirks/tuya/sm0202_motion.py:29]: This can be removed now

**Review by TheJulianJES — APPROVED** [review]
LGTM

File: zhaquirks/tuya/sm0202_motion.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        ENDPOINTS: {
+            1: {
+                PROFILE_ID: zha.PROFILE_ID,
+                DEVICE_TYPE: zha.DeviceType.IAS_ZONE,
+                INPUT_CLUSTERS: [
+                    Basic.cluster_id,
+                    PowerConfiguration.cluster_id,
+                    Identify.cluster_id,
+                    IasZone.cluster_id,
+                    0xEEFF,  # Unknown
```
Comment (TheJulianJES) [inline zhaquirks/tuya/sm0202_motion.py:43]: Ah, actually. I think we should still add this to the `replacement`?
(But I guess it doesn't really make a difference)

Comment (Tagnard) [reply zhaquirks/tuya/sm0202_motion.py:43]: Like this?

```
replacement = {
        ENDPOINTS: {
            1: {
                PROFILE_ID: zha.PROFILE_ID,
                DEVICE_TYPE: zha.DeviceType.IAS_ZONE,
                INPUT_CLUSTERS: [
                    Basic.cluster_id,
                    PowerConfiguration.cluster_id,
                    Identify.cluster_id,
                    IasZone.cluster_id,
                    0xEEFF,  # Unknown
                    MotionCluster,
                ],
                OUTPUT_CLUSTERS: [Ota.cluster_id],
            },
        },
    }
```

Comment (TheJulianJES) [reply zhaquirks/tuya/sm0202_motion.py:43]: No, I meant that the unknown cluster ID `0xEEFF` could also be added to the lower replacement part (below where you use `MotionCluster`).
But it's probably fine as-is.

**Pull request merged**